sdk/core/Azure.Core.Amqp/src/Shared/AmqpAnnotatedMessageConverter.cs

```diff
@@ -659,7 +659,7 @@ private static bool TryGetValueBody(AmqpMessage source, out AmqpMessageBody? val
         ///
         /// <returns><c>true</c> if a message property value was able to be created; otherwise, <c>false</c>.</returns>
         ///
-        private static bool TryCreateNetPropertyFromAmqpProperty(
+        public static bool TryCreateNetPropertyFromAmqpProperty(
```

Comment:
These being bumped to `public` is just for testing?

Reply:
It will also be needed by the isolated worker extension.